### PR TITLE
FFM-4195: New erlang sdk reference

### PR DESCRIPTION
**Pull request by StrawberryPad**
* This has been tech reviewed
* [**Link to page in local build**](http://localhost:3000/docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference)

### CODE REVIEW

**Review by SmritiSatya — DISMISSED** [review]

**Review by erdirowlands — APPROVED** [review]

File: docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+      attributes => #{email => <<"demo@harness.io">>}
+    },
+    FlagIdentifier = "harnessappdemodarkmode",
+    Result = cfclient:bool_variation(FlagIdentifier, Target, false),
+    logger:info("Varaion for Flag ~p witih Target ~p is: ~p~n", [FlagIdentifier, maps:get(identifier, Target), Result]),
+    timer:sleep(10000),
+    get_flag_loop().
+
+  ```
+
```
Comment (erdirowlands) [inline docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md:577]:
```suggestion
  * You can also find a sample Erlang apllication that uses the Erlang SDK in [GitHub](https://github.com/harness-apps/ff-erlang-server-sample)
  
```

Last minute addition @StrawberryPad !  See our offline conversation, the TLDR is I was able to place this repo in a public location in the "harness-apps" github accoubt.

**Review by erdirowlands — DISMISSED** [review]

**Review by erdirowlands — CHANGES_REQUESTED** [review]
Another addition there which I missed - see suggested change

File: docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+  
+  %% Close a named instance
+  cfclient:close(instance_name_1).
+  ```
+
+* **For Elixir applications:**
+
+  ```
+  # Close the default instance
+  :cfclient:close()
```
Comment (erdirowlands) [inline docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md:286]:
```suggestion
  :cfclient.close()
```

Syntax error

File: docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+  ```
+
+* **For Elixir applications:**
+
+  ```
+  # Close the default instance
+  :cfclient.close()
+  
+  # Close a named instance
+  :cfclient:close(:instance_name_1)
```
Comment (erdirowlands) [inline docs/feature-flags/ff-sdks/server-sdks/erlang-sdk-reference.md:289]:
```suggestion
  :cfclient.close(:instance_name_1)
```

**Pull request merged**